mkdocs.yml

```diff
@@ -1,14 +1,12 @@
-### Site metadata ###
-
+# Existing mkdocs.yml content
 site_name: Algoritmekader [concept]
 site_description: Concept van het algoritmekader. Het doel daarvan is om overheden op praktische wijze te ondersteunen, zodat zij op een wettige en ethisch verantwoorde wijze algoritmes en AI-systemen gebruiken.
-watch:
```

Comment:
deze is denk ik nog wel nodig voor de tooltips met definities? Of werkt dat nu op een andere manier?